src/WebJobs.Script.WebHost/Management/LinuxSpecialization/PackageDownloadHandler.cs

```diff
@@ -214,5 +229,57 @@ private async Task<bool> IsResourceAccessibleWithoutAuthorization(string resourc
                 return false;
             }
         }
+
+        private string CopyPackageFile(IFileSystem fileSystem = null)
+        {
+            fileSystem ??= FileUtility.Instance;
+            var packageFolderPath = _environment.GetSitePackagesPath();
+
+            if (!fileSystem.Directory.Exists(packageFolderPath))
+            {
+                CopyPackageFileFailed($"{nameof(CopyPackageFile)} failed. {ScriptConstants.SitePackagesFolderName} folder doesn't exist at {packageFolderPath}.");
+            }
+
+            var packageNameTxtPath = _environment.GetSitePackageNameTxtPath();
+            if (!fileSystem.File.Exists(packageNameTxtPath))
+            {
+                CopyPackageFileFailed($"{nameof(CopyPackageFile)} failed. {ScriptConstants.SitePackageNameTxtFileName} doesn't exist at {packageNameTxtPath}.");
+            }
+
+            var packageFileName = fileSystem.File.ReadAllText(packageNameTxtPath);
+
+            if (string.IsNullOrEmpty(packageFileName))
+            {
+                CopyPackageFileFailed($"{nameof(CopyPackageFile)} failed. {ScriptConstants.SitePackageNameTxtFileName} is empty at {packageNameTxtPath}.");
```

Comment:
>{nameof(CopyPackageFile)} [](http://example.com/codeflow?start=40&length=25)

we dont need this prefix, the source column should be sufficient to know where this is logged